### ISSUE TITLE
Color contrast tool `ResourceCard`: missing link to tool

### ISSUE THREAD

**Issue by techtolentino** [carbon-design-system/carbon-website#2990]
## Detailed description

The Color contrast tool `ResourceCard` is missing its `href`

## Steps to reproduce the issue

1. Visit the [Tools section](https://carbondesignsystem.com/guidelines/accessibility/overview/#tools) under the Accessibility guidelines
2. Click on the Color contrast card
3. See error

![Screen Shot 2022-06-14 at 10 51 58 AM (2)](https://user-images.githubusercontent.com/6276074/173656593-64af5d0b-d580-4a49-b02d-46cff23c4ef1.png)
